### ISSUE TITLE
React Native 0.71.1 android, not renaming package in release folder

### ISSUE THREAD

**Issue by knipferrc** [junedomingo/react-native-rename#192]
**Describe the bug**
Running rename on a new 0.71.1 react native project, the ReactNativeFlipper.java file does not get its package name updated and associated folder changes.

**To Reproduce**
1. Create new react-native project
2. Run `npx react-native-rename "AwesomeProject" -b "com.new.bundle"`

**Expected behavior**
Expected to see the ReactNativeFlipper.java file to get its package name updated along with the directory structure.

**Environment (please complete the following information):**
 - operating system: MacOS
 - npm version: 9.3.1
 - node version: 19.5.1
 - react-native-rename version: latest
 - react-native version: 0.71.1

**Comment by junedomingo** [comment]
Hi @knipferrc , the new path for ReactNativeFlipper.java has been added since [v3.2.9](https://github.com/junedomingo/react-native-rename/releases/tag/v3.2.9).

also the previous folder "android/app/src/release/java/com/awesomeproject" is currently not being deleted but, every content from there should be moved to "android/app/src/release/java/com/new/bundle".
<img width="464" alt="Screenshot 2023-01-31 at 9 43 49 AM" src="https://user-images.githubusercontent.com/5106887/215638240-ad92bf97-d689-4d7a-ba06-740d5c065fca.png">

**Comment by knipferrc** [comment]
@junedomingo Yeah I saw support was added in 3.2.9, but for some reason for me, its ignoring the release folder. Created a fresh react-native project and this is whats happening for me. It renames everything else but not the release folder. I had to add package name to the AndroidManifest.xml file as well for it to run. Maybe i'm doing something incorrectly? 

![image](https://user-images.githubusercontent.com/5757096/215760596-f05f752d-a1c5-4d32-a0a1-75afd2f34bcd.png)

**Comment by junedomingo** [comment]
I think you're using the old version. try `npx react-native-rename@latest "AwesomeProject" -b "com.test.app"`

**Comment by knipferrc** [comment]
ahh ok yeah that seems to be the case. After adding the latest tag on there it worked. I think the reason its not working in my other app is the app name is only 3 characters which seems to throw and error since the minimum character count is 4. Any way around that?

**Comment by junedomingo** [comment]
For names, the minimum is 1 and the maximum is 30.
If your app name only contains < 4 valid characters, it will ask you to provide `pathContentStr` to be used in renaming the folders, files, and their contents.

What you can do is something like this `npx react-native-rename@latest "Awe" -p "AwesomeApp" -b "com.awesome.app"`.

`pathContentStr` - a string that is close to your app name

**Comment by knipferrc** [comment]
When passing a content string though it looks like it’s renaming it to that string. Is there a way to keep the name at the 3 valid characters as I would prefer not to change it. Really I just need to be able to change the bundle ids and not the name

**Comment by junedomingo** [comment]
@knipferrc could you provide an example? name and bundleId? (with exact number of characters)

**Comment by junedomingo** [comment]
```bash
npx react-native-rename@latest "dem" -p "demo app"
ios/demoapp RENAMED
ios/demoapp.xcworkspace RENAMED
ios/demoappTests RENAMED
ios/demoappTests/demoappTests.m RENAMED
```

above, it uses a 3-character name "dem", if your run it, your app should be named "dem"

**Comment by junedomingo** [comment]
I will close this now as the real issue was already resolved.
